packages/python/readme_metrics/PayloadBuilder.py

```diff
@@ -224,6 +220,9 @@ def _build_request_payload(self, request) -> dict:
             "bodySize": -1,
         }
 
+        if "Authorization" in headers:
```

Comment:
at this point, if there is an `Authorization` header, won't there already be an unmasked Authorization header in `payload["headers"]`?

Reply:
That would be my expectation too - does `.append` overwrite the existing header here, or do we need to forcefully delete it?